nevergrad/functions/base.py

```diff
@@ -102,6 +103,9 @@ def parametrization(self) -> p.Parameter:
     def parametrization(self, parametrization: p.Parameter) -> None:
         self._parametrization = parametrization
         self._parametrization.freeze()
+        # pylint: disable=pointless-statement
+        self._parametrization.random_state  # force initialization for synchronization of random state
```

Comment:
this makes sure the parametrization has it's random_state initialized at the very beginning. but it's unclear why it would be needed now and not before